src/Commands/Base.js

```diff
@@ -43,9 +43,17 @@ class Base extends Command {
    */
   * _checkRequirements () {
     const nodeVersion = process.version
-    const npmVersion = yield pify(exec)('npm -v')
+    let npmVersion
     let error = false
 
+    // retrieve npm version.
+    pify(exec)('npm -v').then(version => {
```

Comment:
I'd prefer to use `yield pify(...)` inside a try/catch block instead of using Promise directly here.